rest/data/repotracker.go

```diff
@@ -46,40 +46,55 @@ func (c *RepoTrackerConnector) TriggerRepotracker(q amboy.Queue, msgID string, e
 		}))
 		return err
 	}
-	fields := message.Fields{
-		"source":      "github hook",
-		"msg_id":      msgID,
-		"event":       "push",
-		"owner":       *event.Repo.Owner.Name,
-		"repo":        *event.Repo.Name,
-		"ref":         *event.Ref,
-		"project_ref": ref.Identifier,
-		"message":     "actionable push acknowledged",
-	}
-	if !ref.TracksPushEvents || !ref.Enabled {
-		fields["tracks_push_events"] = ref.TracksPushEvents
-		fields["enabled"] = ref.Enabled
-		fields["message"] = "unactionable push acknowledged"
-		grip.Info(fields)
-		return nil
+
+	actionable := []string{}
+	unactionable := []string{}
+	failed := []string{}
+	catcher := grip.NewSimpleCatcher()
+	for i := range refs {
+		if !refs[i].TracksPushEvents || !refs[i].Enabled {
+			unactionable = append(unactionable, refs[i].Identifier)
+			continue
+		}
+
+		if err := q.Put(units.NewRepotrackerJob(fmt.Sprintf("github-push-%s", msgID), refs[i].Identifier)); err != nil {
+			catcher.Add(errors.Errorf("failed to add repotracker job to queue for project: '%s'", refs[i].Identifier))
+			failed = append(failed, refs[i].Identifier)
+
+		} else {
+			actionable = append(actionable, refs[i].Identifier)
```

Comment:
I think you can unindent this and remove the else.

Reply:
I should've called that "succeeded"